charts/posthog/values.yaml

```diff
@@ -481,7 +481,14 @@ kafka:
     # - Enable data persistence using PVC.
     enabled: true
     # -- PVC Storage Request for Kafka data volume.
-    size: 20Gi
+    size: 80Gi
+  # -- A size-based retention policy for logs
+  # -- Should be less than kafka.persistence.size, minimum 1GB
+  logRetentionBytes: _2_333_333_333
+  # -- The minimum age of a log file to be eligible for deletion due to age
+  logRetentionHours: 24
```

Comment:
Problem to be solved here is ensuring that users don't run out of disk. I think we aren't really sure whether this is max(logRetentionBytes, logRetentionHours) or min.